src/plugins/data/public/query/query_string/dataset_service/lib/index_type.test.ts

```diff
@@ -104,30 +109,54 @@ describe('indexTypeConfig', () => {
     expect(result.hasNext).toBe(true);
   });
 
-  test('should filter out data sources with versions lower than 1.0.0', async () => {
-    mockSavedObjectsClient.find = jest.fn().mockResolvedValue({
-      savedObjects: [
-        { id: 'ds1', attributes: { title: 'DataSource 1', dataSourceVersion: '1.0' } },
-        {
-          id: 'ds2',
-          attributes: { title: 'DataSource 2', dataSourceVersion: '' },
+  describe('fetchIndices', () => {
+    test('should extract index names correctly from different formats', async () => {
+      const mockResponse = {
+        rawResponse: {
+          aggregations: {
+            indices: {
+              buckets: [
+                // Serverless format with TIMESERIES
```

Comment:
nit: i think serverless is a product. probably shouldn't mention here

Reply:
will remove this comment